### PR TITLE
Use the right attributes separator into the cart

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Use the right attributes separator into the cart
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-2205
| How to test?  |

### CODE REVIEW

File: classes/Cart.php [hunk 1]
```diff
@@ -952,6 +952,7 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
         }
 
         $pa_implode = array();
+        $separator = Tools::getValue('PS_ATTRIBUTE_ANCHOR_SEPARATOR', '-');
```
Comment (thierrymarianne) [inline classes/Cart.php]: Shouldn't `PS_ATTRIBUTE_ANCHOR_SEPARATOR` be passed to `Configuration::get`?

Comment (maximebiloe) [inline classes/Cart.php]: oops 😵 🚪

**Comment by AntoineMille** [comment]
Thanks @maximebiloe, It's good for me

**Comment by aleeks** [comment]
Thank you @maximebiloe

**Pull request merged**